R/FilterStateRange.R

```diff
@@ -456,10 +456,29 @@ RangeFilterState <- R6::R6Class( # nolint
       tagList(
         div(
           class = "choices_state",
+          tags$head(tags$script(
+            HTML(
+              '$(document).ready(function() {
+                      $("[data-toggle=\'popover\']").popover();});'
+            )
+          )),
           div(
-            actionLink(ns("plotly_info"), label = NULL, icon = icon("question-circle")),
+            actionLink(
+              ns("plotly_info"),
+              label = NULL,
+              icon = icon("question-circle"),
+              "data-toggle" = "popover",
+              "data-html" = "true",
+              "data-placement" = "left",
+              "data-trigger" = "hover",
+              "data-content" = '<p>Drag vertical lines to set selection.<br>
+                              Drag across plot to zoom in.<br>
+                              Drag axis to pan.<br>
+                              Double click to zoom out.</p>'
```

Comment:
```suggestion
              "data-content" = '<p>
                                <h5> Plot actions </h5>
                                Drag vertical lines to set selection.<br>
                                Drag across plot to zoom in.<br>
                                Drag axis to pan.<br>
                                Double click to zoom out.
                                </p>'
```

Can you remove the excess top and bottom padding from the popover?